Package.swift

```diff
@@ -4,10 +4,16 @@ import Foundation
 import PackageDescription
 
 /// Swift settings that should be applied to every Swift target.
-let globalSwiftSettings: [SwiftSetting] = [
-  .enableUpcomingFeature("InternalImportsByDefault"),
-  .enableUpcomingFeature("MemberImportVisibility"),
-]
+var globalSwiftSettings: [SwiftSetting] {
+  var result: [SwiftSetting] = [
+    .enableUpcomingFeature("InternalImportsByDefault"),
+    .enableUpcomingFeature("MemberImportVisibility"),
+  ]
+  if noSwiftPMDependency {
```

Comment:
I was going to ask where this was set, but we apparently had it already 😅. The double negative is a bit weird, but I suppose having `SWIFTPM_DEPENDENCY` would also be a bit weird, so maybe the double negative is the lesser of two evils.